### DISCUSSION TITLE
Import products from XML

### DISCUSSION THREAD

**Discussion by Twilight-Computer** [givanz/Vvveb#111] [Ideas]
I have feeds from serveral suppliers - it would be a nice function to be able to import / update products from XML

**Comment by givanz** [comment]
Vvveb has XML import feature in `admin > tools > import/export`.

You can export current products by selecting at least `product` and `product_content` tables to see the format that it uses, add more product tables like attribute/options etc if you need the format for other tables.

![export](https://github.com/givanz/Vvveb/assets/302738/06d2cf5b-ea15-4b49-8ff7-2181915396f7)

The XML format is a simple structure of the table columns and row data, you can use an XML tool to map the nodes from your XML feeds to the Vvveb format and generate XMLs to import.
